<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TalkDialog</class>
 <widget class="QDialog" name="TalkDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>755</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">#dialogueFrame {
	border-image: url(&quot;:/images/parchment&quot;) 0 0 0 0 stretch stretch;
}

#characterTitle {
	background-color: rgba(0, 0, 0, 200);
	color: rgba(255, 255, 255, 255);
	font-weight: bold;
	padding: 2px;
}

#dialogueSelector {
	background: brown;
}

#characterSelector {
	background: rgba(0, 0, 0, 150);
	color: #ffffff;
	font-weight: bold;
}

#dialogueList {
	background: rgba(0, 0, 0, 150);
	color: #ffffff;
	font-weight: bold;
}

#firstOption, #secondOption, #thirdOption {
	background: rgba(255, 255, 255, 50);
	border: rgba(0, 0, 0, 50);
    border-style: solid;
    border-width: 5px;
	font-weight: bold;
	font-style: italic;
}

#Speak, #Done {
	background-color: rgba(255, 255, 255, 200);
	border: rgba(0, 0, 0, 100);
    border-style: inset;
    border-width: 5px;
	color: black;
	font-weight: bold;
	border-radius: 15px;
}

#Speak:hover, #Done:hover {
	background-color: rgba(155, 155, 155, 200);
	color: white;
}</string>
  </property>
  <widget class="QFrame" name="dialogueFrame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>755</width>
     <height>521</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="Speak">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>400</y>
      <width>93</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Say</string>
    </property>
   </widget>
   <widget class="QListWidget" name="dialogueList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>721</width>
      <height>221</height>
     </rect>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="characterSelector">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>240</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="secondOption">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>350</y>
      <width>171</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>Option 2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Done">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>460</y>
      <width>93</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Done</string>
    </property>
   </widget>
   <widget class="QDial" name="dialogueSelector">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>360</y>
      <width>71</width>
      <height>64</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="maximum">
     <number>2</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="tracking">
     <bool>true</bool>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="invertedControls">
     <bool>false</bool>
    </property>
    <property name="notchesVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="firstOption">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>171</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>Option 1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="thirdOption">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>381</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Option 3</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="characterTitle">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Character</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
